watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/dye_desalination/dye_desalination.py

```diff
@@ -148,71 +215,216 @@ def eq_flow_mass_comp(blk, j):
                     == blk.properties_out[0].flow_mass_phase_comp["Liq", "H2O"]
                 )
 
+    elif include_dewatering and include_gac:
+        raise TypeError("This system cannot have both dewatering and GAC units.")
```

Comment:
Is this conceptually it should not have both, or that it's just not implemented in the model?

Reply:
I don't think it would make sense conceptually to have both since the purpose behind including these unit models was to remove water from/further concentrate the dye waste stream. If we had these two units in series, the second unit would be redundant and not do very much.